### PR TITLE
feat: project cards onClick direct to repo #34

### LINKED ISSUE #34: Suggestion: Project cards should have a link to the repo
I think the project cards should have a link to the repo for more information.

### LINKED ISSUE #40: Cursor over card should change
If we are to link to the repo via the card the cursor should change to a pointer.

Add the style:
```css
cursor:pointer
```

### PR DESCRIPTION
**Pull request by AllanRegush**
closes #34 
closes #40

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,6 +51,7 @@ <h2 class="text-center text-3xl font-semibold text-indigo-100">
 
         <div
           class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 gap-4"
+          onclick="window.open('https://github.com/EddieJaoudeCommunity/EddieBot','_blank');"
         >
           <div
```
Comment (alrifay) [inline index.html:56]: I think you misplaced the `onclick` event in line **54**, it should be on the `div` in line *56*

File: index.html [hunk 2]
```diff
@@ -70,6 +71,7 @@ <h3 class="mt-3 ml-3 font-bold text-indigo-100 text-xl mb-2">
 
           <div
```
Comment (alrifay) [inline index.html:72]: I think it's better to replace the `div` tag with `a` tag in all cards and it will fix the cursor style

Comment (eddiejaoude) [reply index.html:72]: I think for accessibility using `a` tag is better too 👍

**Comment by AllanRegush** [comment]
closes #40

**Comment by eddiejaoude** [comment]
I think `closes #40` needs to be in the description, you can have do `closes` in the description

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**